src/Surfnet/StepupSelfService/SelfServiceBundle/Controller/RemoteVetting/MockConfiguration.php

```diff
@@ -0,0 +1,85 @@
+<?php
+/**
+ * Copyright 2019 SURFnet B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Surfnet\StepupSelfService\SelfServiceBundle\Controller\RemoteVetting;
+
+final class MockConfiguration
+{
+    private $identityProviderEntityId;
+    private $serviceProviderEntityId;
+    private $publicKeyCertData;
+    private $privateKeyPem;
+
+    /**
+     * @param string $identityProviderEntityId
+     * @param string $serviceProviderEntityId
+     * @param string $privateKeyPath
+     * @param string $publicCertPath
+     */
+    public function __construct(
+        $identityProviderEntityId,
+        $serviceProviderEntityId,
+        $privateKeyPath,
+        $publicCertPath
+    ) {
+
+        if (!is_file($privateKeyPath)) {
+            throw new \InvalidArgumentException('Unable to find private key: '. $privateKeyPath);
```

Comment:
Enabled Travis for Gitflow branches so this now gets detected with Phpmd. I don't see the added benefit of creating a custom exception class and because this code wouldn't be used in production but only in dev and test mode I think this general exception will outweigh the introduction and maintenance of a class not used in production.